src/components/Text.tsx

```diff
@@ -387,8 +387,8 @@ export function SphereText({ children }: ChildrenProp) {
   return <p className={sphereText}>{children}</p>
 }
 
-const sectionTitle = classnames(fontWeight('font-bold'), fontSize('text-sm'))
 export function SectionTitle({ children }: ChildrenProp) {
+  const sectionTitle = fontSize('text-sm')
```

Comment:
Now we have section title (more like a subtitle) not bold, because the title (`NFTs` should be bold):
![image](https://user-images.githubusercontent.com/49658988/187157852-3b765d16-0de4-4b19-8c91-885976cc9c07.png)